src/stormpath.registration.js

```diff
@@ -45,7 +45,11 @@ angular.module('stormpath')
             password: $scope.formModel.password
           })
           .then(function(){
-            if($scope.postLoginPath){
+            var $state = $injector.get('$state');
```

Comment:
This line will throw an error if UI Router is not used. Use `$injector.has()` to check if the service exist:

```javascript
var $state = $injector.has('$state') ? $injector.get('$state') : null;
```